Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imdining",category=['you'],prompt="I'm Dining",pool=True,unlocked=True))
+if seen_event(monika_imdining) = False:
```

Comment:
Could I ask what this does?

Reply:
Welp, if you had to ask, then that means I did it wrong. 
It was supposed to initiate that bit of dialogue for the first time only. Lol